Applications/Help/main.cpp

```diff
@@ -51,14 +51,14 @@
 
 int main(int argc, char* argv[])
 {
-    if (pledge("stdio shared_buffer accept rpath unix cpath fattr", nullptr) < 0) {
+    if (pledge("stdio shared_buffer accept exec rpath unix cpath fattr", nullptr) < 0) {
         perror("pledge");
         return 1;
     }
 
     GUI::Application app(argc, argv);
 
-    if (pledge("stdio shared_buffer accept rpath", nullptr) < 0) {
+    if (pledge("stdio shared_buffer accept exec rpath", nullptr) < 0) {
```

Comment:
could we use LibDesktop's `open(URL)` instead of running the browser directly?

Reply:
Good idea, done!